packages/sdk/src/lib.rs

```diff
@@ -13,6 +13,11 @@ pub use cw_storage_plus;
 #[cfg(feature = "neutron")]
 pub use neutron_sdk;
 
+pub const RELEASE_VERSION: &str = env!(
+    "RELEASE_VERSION",
+    r#"No release version provided as an environment variable! Please set "RELEASE_VERSION" environment variable!"#,
+);
+
```

Comment:
There is no point in keeping this constant in the `sdk` library. Is not `versioning` a better alternative?